modules/rosa-install-logging-addon.adoc

```diff
@@ -21,6 +21,13 @@ For `<cluster_name>`, enter the name of your cluster.
 
 . When prompted, accept the default `yes` to install the `cluster-logging-operator`.
 . When prompted, accept the default `yes` to install the optional Amazon CloudWatch log forwarding add-on or enter `no` to decline the installation of this add-on.
++
+[NOTE]
+====
+You can install the AWS CloudWatch service at any time after installing `cluster-logging-operator` through the OpenShift Cluster Manager (OCM) console:
+. Click a cluster in OCM, and click the *Add-ons* tab.
+. Click the CloudWatch service, and click *Install*.
+====
```

Comment:
It looks like the procedure steps in this note aren't rendering as expected in the preview.

Reply:
One more wording consideration to clarify what is being installed through the OCM:

> After installing the `cluster-logging-operator`, you can install the AWS CloudWatch service at any time through the OpenShift Cluster Manager (OCM) console from the cluster's *Add-ons* tab.

Reply:
I see what you're saying, but in context of the rest of the procedure, that's not quite clear enough either. See what you think about the new wording. Hopefully it's really clear now.

Reply:
Looks great!